ui/src/message_popup/author/questions_page_test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from 'chai';
+
+import QuestionsPage from './questions_page.jsx';
+import QuestionButton from './question_button.jsx';
+import ArchivedQuestionButton from './archived_question_button.jsx';
+import {testQuestions} from '../test_fixtures.js';
+
+function testProps(props) {
+  return {
+    allQuestions: testQuestions,
+    onReloadQuestions: sinon.spy(),
+    loaded: true,
+    ...props
+  };
+}
+
+describe('<QuestionsPage />', ()=>{
+  it('renders current questions when loaded', ()=>{
+    const props = testProps();
+    const wrapper = shallow(<QuestionsPage {...props} />);
+    expect(wrapper.find(QuestionButton)).to.not.have.length(0);
```

Comment:
Would it make sense to assert on how many questions there are, or that the number rendered matches the number in `testQuestions `?